src/api/contract/kubectl/v1.ts

```diff
@@ -7,6 +7,7 @@ import * as vscode from 'vscode';
 export interface KubectlV1 {
     invokeCommand(command: string): Promise<KubectlV1.ShellResult | undefined>;
     portForward(podName: string, podNamespace: string | undefined, localPort: number, remotePort: number, options?: KubectlV1.PortForwardOptions): Promise<vscode.Disposable | undefined>;
+    getKubeconfigPath(): string;
```

Comment:
As mentioned in email, I'm anxious even about 'add a member' changes to existing APIs.  A user might pin to an old version of the k8s extension, say because I introduced a bug that affects them in a more recent one (oh, say it ain't so); in which case consumers of KubectlV1 would expect the new method to be available but get an error at runtime.  We should either roll forward to a `KubectlV1_1` interface or put it on a new 'configuration' or 'environment' interface.

Reply:
Yes! Moved to its own api.